src/main/telemetry/crsf.c

```diff
@@ -255,7 +255,7 @@ void crsfFrameDeviceInfo(sbuf_t *dst) {
     for (unsigned int ii=0; ii<12; ii++) {
         sbufWriteU8(dst, 0x00);
     }
-    sbufWriteU8(dst, CRSF_DEVICEINFO_PARAMETER_COUNT);
+    sbufWriteU8(dst, 0);
```

Comment:
Even if it's 0, using a define for it instead of a magic number will make the data format self documenting.

Reply:
Done.